spec/button_spec.rb

```diff
@@ -123,29 +202,65 @@
           end
         end
 
+        it 'raises an ArgumentError when there is no button with that id' do
+          expect{ ShareProgress::Button.find(999999999)}.to raise_error ArgumentError
```

Comment:
@EricBoersma luckily the request is mocked. It'll always return a `404` thanks to webmock/vcr:

https://github.com/SumOfUs/share_progress/blob/buttons/spec/vcr/cassettes/ShareProgress_Button/class_methods/find/receiving_data/raises_an_ArgumentError_when_there_is_no_button_with_that_id.yml#L5